app/main.py

```diff
@@ -1,6 +1,19 @@
-from typing import Callable
+from typing import Callable, Any
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_dict = {}
+
+    def inner(*args, **kwargs) -> Any:
+        func_name = f"{func.__name__}"
```

Comment:
Using an f-string in this case, doesn't make sense because `func.__name__` is already a string. 
```suggestion
        func_name = func.__name__
```